### ISSUE TITLE
Inconsistent description of chapters

### ISSUE THREAD

**Issue by wgropp** [mpi-forum/mpi-issues#563]
# Problem

In the intro chapter, when each of the chapters is briefly described, the collective chapter description includes 

> With \MPIII/, the semantics of collective
communication
was extended
to include inter-communicators.
It also adds
two new collective operations.
\MPIIII/ adds nonblocking collective operations.
\MPIIV/ adds persistent nonblocking collective operations.

None of the other chapters describes the history of the contents, and some are quite general, for example, the description of the process topologies chapter. 

The Dynamic process chapter does have a "MPI-4 adds a new Sessions Model". Like the above, this could be deleted; the text describing the Dynamic chapter is correct and similar in detail to the others without this mention.

The Removed Interfaces chapter includes 

> These were
deprecated in \MPIII/, and the \MPI/ Forum decided to remove these from
the \MPIIII/ standard.

This is again historical; I suggest dropping this text.  The description remains correct, and won't be made incorrect should we remove any interfaces in a subsequent version of MPI.
# Proposal

Remove the above mentioned text. An alternative is to add a short note that this includes nonblocking and persistant nonblocking.  However, as the description of the point-to-point chapter does not mention either of those, my preference is just to drop the quoted text.

# Changes to the Text

As noted above, delete the historical comment on collectives, the addition of Sessions, and the comment on removing interfaces in MPI 3.

# Impact on Implementations

None

# Impact on Users

None

# References and Pull Requests

N/A

**Comment by wesbland** [comment]
I would suggest that the intro chapter committee takes this one and gets feedback from the relevant other chapter committees to confirm that the text is good.

**Comment by wgropp** [comment]
I propose to remove the text as above unless there is an objection.

**Comment by schulzm** [comment]
Makes sense to me - otherwise we would have to add text like this for all chapters and that could arbitrarily long. When we delete the text above, we may also want to consider deleting the historical comment for the tools chapter.

**Comment by RolfRabenseifner** [comment]
> I propose to remove the text as above unless there is an objection.

I agree.

Tools chapter: here, I would substitute the second sentence by "It consists of a profiling and a tools information interface."
because this chapter is really spilt into these two very different parts.

**Comment by wesbland** [comment]
This wasn't read at the December 2022 meeting. The last opportunity for MPI 4.1 is to have it ready at the March 2022 meeting (and it needs to "pass" the reading).

@wgropp / @schulzm Are you (or is someone else) planning to push this forward or should we move it out of the plan for MPI 4.1?

**Comment by wgropp** [comment]
I'll try and put this together for March.

**Comment by wgropp** [comment]
On the tools chapter and the part that comments on Section 15.2 being a chapter in a previous version, I've simply removed that. Note that we don't say anything about removing the datatypes discussion from the point-to-point chapter. 

This is a rare PR that (very slightly) reduces the size of the MPI standard.

**Comment by wgropp** [comment]
I've also removed the second sentence about the External Interfaces chapter, since that mentions threads, and that text has been moved out of this chapter. Keeping this description short future proofs it.

**Comment by wgropp** [comment]
I've added an update to be read in March 23.

**Comment by schulzm** [comment]
If it is ready, should we pull this into the new Feb meeting?

**Comment by wgropp** [comment]
Yes, this is ready and can be read at the earliest opportunity.

**Comment by wgropp** [comment]
Argh. I don't think this got read in Feb. Is it on the list for March?

**Comment by wesbland** [comment]
**TL;DR** - Not yet :)

The only thing that's been announced for March is the email that Rolf sent out a couple of hours ago.

All of the reading announcements need to go to the mailing list so everyone sees them. I don't do any "automatic" announcements unless we don't get to some readings that might have been announced for a previous meeting.

**Comment by RolfRabenseifner** [comment]
There is no pull request.

**Comment by wgropp** [comment]
https://github.com/mpi-forum/mpi-standard/pull/780

**Comment by Wee-Free-Scot** [comment]
> Chapter~\ref{chap:datatypes}, \nameref{chap:datatypes}, defines a method to describe any data layout,
e.g., an array of structures in the memory,
which can be used as message send or receive buffer.

Suggestion:

Chapter~\ref{chap:datatypes}, \nameref{chap:datatypes}, defines a method to describe any data layout,
e.g., an array of structures.

Reasons:
a) "in the memory" excludes in file views.
b) "which can be used as message send or receive buffer" excludes as message buffer for collective operations (input/output != send/receive), as message buffer for a put/get/accumulate operations, and as a data buffer for file read/write operations.
c) removal is a simpler correct change than extending the text to include all the additional usages.
d) shrinking MPI, one line at a time.

**Comment by wgropp** [comment]
I did send a note to the Forum to read this. Did it get scheduled?

**Comment by schulzm** [comment]
Let me check - the agenda is not 100% up to date, yet

**Comment by schulzm** [comment]
yes, I have that on the agenda

**Comment by wgropp** [comment]
Updated the PR to include the improved datatype chapter description.

**Comment by wesbland** [comment]
This passed a [1st vote](https://mpi-forum.org/meetings/2023/05/votes).
<table>
<tr>
    <td> Yes
    <td> No
    <td> Abstain
<tr>
    <td> 28
    <td> 0
    <td> 4
</table>

**Comment by wesbland** [comment]
This passed a [2nd vote](https://mpi-forum.org/meetings/2023/07/votes).
<table>
<tr>
    <td> Yes
    <td> No
    <td> Abstain
<tr>
    <td> 34
    <td> 0
    <td> 0
</table>
